app/views/training/pages/_citation.html.slim

```diff
@@ -1,8 +1,8 @@
-.cdt-citation
+.citation-wrapper
   .citation
```

Comment:
@martikat you can target and style elements purely by having them be wrapped. Many of these definitions can be replaced with elements.

Reply:
This recommendation hasn't been realised yet.

1. you have more classes than you need, you can just style the `dl`, `dt` and `dd` elements in their `div`
2. this styling refers to a citation but doesn't contain a 
    > a quotation from or reference to a book, paper, or author

    Instead, it is for entry metadata, publication timestamps so the use of `citation` is misleading.